mobile_packet_verifier/src/accumulate.rs

```diff
@@ -18,14 +18,12 @@ pub async fn accumulate_sessions(
     verified_data_session_report_sink: &FileSinkClient<VerifiedDataTransferIngestReportV1>,
     curr_file_ts: DateTime<Utc>,
     reports: impl Stream<Item = DataTransferSessionIngestReport>,
-    cbrs_disable_time: DateTime<Utc>,
 ) -> anyhow::Result<()> {
     tokio::pin!(reports);
 
     while let Some(report) = reports.next().await {
-        if report.received_timestamp >= cbrs_disable_time
-            && report.report.data_transfer_usage.radio_access_technology
-                == DataTransferRadioAccessTechnology::Eutran
+        if report.report.data_transfer_usage.radio_access_technology
+            == DataTransferRadioAccessTechnology::Eutran
```

Comment:
Could we put this in a function somewhere, like in the ingester. 
I've had the question before, and I'm sure I'll have it again after I've forgotten of what does `Eutran` mean. And it's not totally clear it means CBRS.

Reply:
This is the only place we use it in such way and I think we don't want to add this to our codebease in near future.
Added comment for this line.